### PR TITLE
feat: make repo releasable

### PR DESCRIPTION
**Pull request by chingor13**

### CODE REVIEW

File: .github/release-please.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+releaseType: java-yoshi
+bumpMinorPreMajor: true
```
Comment (athakor) [inline .github/release-please.yml]: Add newline at end of file

File: .kokoro/continuous/lint.cfg [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+env_vars: {
+  key: "TRAMPOLINE_IMAGE"
+  value: "gcr.io/cloud-devrel-kokoro-resources/java8"
+}
+
+env_vars: {
+  key: "JOB_TYPE"
+  value: "lint"
+}
```
Comment (athakor) [inline .kokoro/continuous/lint.cfg]: same here.

File: .kokoro/nightly/lint.cfg [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+env_vars: {
+  key: "TRAMPOLINE_IMAGE"
+  value: "gcr.io/cloud-devrel-kokoro-resources/java8"
+}
+
+env_vars: {
+  key: "JOB_TYPE"
+  value: "lint"
+}
```
Comment (athakor) [inline .kokoro/nightly/lint.cfg]: same here.

File: .kokoro/presubmit/clirr.cfg [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+env_vars: {
+  key: "TRAMPOLINE_IMAGE"
+  value: "gcr.io/cloud-devrel-kokoro-resources/java8"
+}
+
+env_vars: {
+  key: "JOB_TYPE"
+  value: "clirr"
+}
```
Comment (athakor) [inline .kokoro/presubmit/clirr.cfg]: same here.

File: .kokoro/presubmit/linkage-monitor.cfg [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+# Configure the docker image for kokoro-trampoline.
+env_vars: {
+  key: "TRAMPOLINE_IMAGE"
+  value: "gcr.io/cloud-devrel-kokoro-resources/java8"
+}
+
+env_vars: {
+  key: "TRAMPOLINE_BUILD_FILE"
+  value: "github/java-storage/.kokoro/linkage-monitor.sh"
+}
```
Comment (athakor) [inline .kokoro/presubmit/linkage-monitor.cfg]: same here.

File: .kokoro/presubmit/lint.cfg [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+env_vars: {
+  key: "TRAMPOLINE_IMAGE"
+  value: "gcr.io/cloud-devrel-kokoro-resources/java8"
+}
+
+env_vars: {
+  key: "JOB_TYPE"
+  value: "lint"
+}
```
Comment (athakor) [inline .kokoro/presubmit/lint.cfg]: same here.

File: .kokoro/release/common.sh [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+      <password>${SONATYPE_PASSWORD}</password>
+    </server>
+    <server>
+      <id>sonatype-nexus-snapshots</id>
+      <username>${SONATYPE_USERNAME}</username>
+      <password>${SONATYPE_PASSWORD}</password>
+    </server>
+  </servers>
+</settings>" > $1
+}
```
Comment (athakor) [inline .kokoro/release/common.sh]: same here.

File: .kokoro/release/snapshot.cfg [hunk 8]
```diff
@@ -0,0 +1,6 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+env_vars: {
+  key: "TRAMPOLINE_BUILD_FILE"
+  value: "github/java-storage/.kokoro/release/snapshot.sh"
+}
```
Comment (athakor) [inline .kokoro/release/snapshot.cfg]: same here.

File: .kokoro/release/stage.sh [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+  -Dgpg.executable=gpg \
+  -Dgpg.passphrase=${GPG_PASSPHRASE} \
+  -Dgpg.homedir=${GPG_HOMEDIR}
+
+if [[ -n "${AUTORELEASE_PR}" ]]
+then
+  mvn nexus-staging:release -B \
+    -DperformRelease=true \
+    --settings=settings.xml
+fi
```
Comment (athakor) [inline .kokoro/release/stage.sh]: same here.

File: .repo-metadata.json [hunk 10]
```diff
@@ -2,12 +2,12 @@
...
   "issue_tracker": "https://issuetracker.google.com/savedsearches/559782",
   "release_level": "ga",
   "language": "java",
-  "repo": "googleapis/google-cloud-java",
-  "repo_short": "google-cloud-java",
+  "repo": "googleapis/java-storage",
+  "repo_short": "java-storage",
   "distribution_name": "com.google.cloud:google-cloud-storage",
   "api_id": "storage.googleapis.com"
 }
```
Comment (athakor) [inline .repo-metadata.json]: same here.

File: pom.xml [hunk 11]
```diff
@@ -0,0 +1,330 @@
...
+
+  <profiles>
+    <profile>
+      <id>include-samples</id>
+      <modules>
+        <module>samples</module>
+      </modules>
+    </profile>
+  </profiles>
+</project>
```
Comment (athakor) [inline pom.xml]: same here

File: synth.metadata [hunk 12]
```diff
@@ -0,0 +1,209 @@
...
+      "path": ".github/ISSUE_TEMPLATE/feature_request.md"
+    },
+    {
+      "path": ".github/ISSUE_TEMPLATE/bug_report.md"
+    },
+    {
+      "path": "__pycache__/synth.cpython-36.pyc"
+    }
+  ]
+}
```
Comment (athakor) [inline synth.metadata]: same here.

**Comment by chingor13** [comment]
@athakor These are common templates and should be addressed globally. I opened https://github.com/googleapis/synthtool/issues/357 to track

**Pull request merged**